client/extensions/woocommerce/woocommerce-services/views/subscriptions-usage/style.scss

```diff
@@ -20,7 +20,43 @@
 		}
 
 		.subscriptions-usage__list-item-usage {
+			display: flex;
+			flex-flow: column;
+			justify-content: center;
+			text-align: center;
 			width: 30%;
+
+			@include breakpoint( '>480px' ) {
+				flex-flow: row;
+				justify-content: flex-start;
+				text-align: left;
+			}
+
+			& > .subscriptions-usage__list-item-usage-icon {
+				display: none;
+			}
+
+			&.is-over-limit {
+				& > .subscriptions-usage__list-item-usage-numbers {
+					color: var( --color-red-light );
```

Comment:
On the PR there is a comment about not using `var( --color-error )` and using `--color-red` instead.
I think it is to be closer to the mockup but I am not sure. Looks ok though.